core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci

```diff
@@ -61,3 +61,5 @@ gcr.io/istio-testing/build-tools:release-1.14-204316d1ee4013432e0b35d8ceba3a943e
 quay.io/maistra-dev/maistra-proxy-builder:2.0 registry.ci.openshift.org/ci/maistra-proxy-builder:2.0
 quay.io/maistra-dev/maistra-proxy-builder:2.1 registry.ci.openshift.org/ci/maistra-proxy-builder:2.1
 quay.io/almartin/odf-console-ci-runner:latest registry.ci.openshift.org/ci/odf-console-ci-runner:latest
+quay.io/strimzi-qe/strimzi-tests:release-0.29.x registry.ci.openshift.org/ci/strimzi-tests:release-0.29.x
```

Comment:
While I see the value in pulling in specific image versions here (`release-0.xx.x`), I am just curious if there is possibly a different tag we could push these images to that would cause less maintenance on PQE's end. They will have to update these images every time it changes in this format.

If the image version won't need to be changed often, it should be fine. But, it may be worth just pushing the images with a `ocp-ci-interop` tag or something that would help pull these images with less maintenance.

Reply:
Addressed these questions in this thread
https://redhat-internal.slack.com/archives/C04QDE5TK1C/p1679413818120849?thread_ts=1678722379.723629&cid=C04QDE5TK1C